openhim-mapping-mediator/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+
+services:
+  openhim-mapping-mediator:
+    image: jembi/openhim-mediator-mapping
+    environment:
+      OPENHIM_REGISTER: "true"
```

Comment:
These should all be variables that default to this value in the package metadata file, please.

Reply:
cool fixed in [695643e](https://github.com/jembi/platform/pull/193/commits/695643ea720c65f3d5bb34d7cf4e430518dbaf71)